src/flows/components/header/index.tsx

```diff
@@ -153,7 +154,7 @@ const FlowHeader: FC = () => {
       try {
         const response = await postNotebooksVersion({id: flow.id})
 
-        if (response.status !== 204) {
+        if (response.status < 200 && response.status > 299) {
```

Comment:
not a fan of this, but we can deal with the API quirk of it returning a 200 later

Reply:
Yeah, why are you still feeling like pushing this to prod?